CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.3.0 - 2022-07-12
+
+### Added
+- New `structure` option for specifying `legacy` nested structure for antora docs, or `default` file structure.
```

Comment:
```suggestion
- New `structure` option for specifying `legacy` nested structure for antora docs.
```
The last part here just didn't make sense. Feel free to come up with a different solution though.

Reply:
Happy with yours